src/components/StateEligibility.js

```diff
@@ -100,16 +100,53 @@ const criteriaGroups = [
             },
         ],
     },
-    /* TODO - remove the following div after March 11, and update link to be appropriate link from https://www.mass.gov/covid-19-vaccine */
+    // -------------------------------------------------------------------------
     {
-        endDate: "2021-03-11T00:00:00-05:00", // current timezone offset is at the end
-        title: "Eligible to sign up starting March 11: ",
+        endDate: "2021-04-19T00:00:00-04:00", // current timezone offset is at the end
```

Comment:
Why will this have an `endDate` of 4/19? Maybe I'm forgetting what `endDate` means. But shouldn't in be 3/22? Same for the April 5 `endDate` below.

Reply:
This is a fast fix.  I will address all of the end dates in another issue

Reply:
ok. maybe you can remove the `3-11` startDate above then too